src/main/java/graphql/GraphQL.java

```diff
@@ -178,11 +179,11 @@ public GraphQL build() {
     }
 
     /**
+     * Executes the specified graphql query/mutation/subscription
+     *
      * @param query the query/mutation/subscription
      * @return result including errors
-     * @deprecated Use {@link #execute(ExecutionInput)}
      */
-    @Deprecated
     public ExecutionResult execute(String query) {
```

Comment:
This is one contentious aspect of this PR.

The most basic way to execute a query (just the string) was deprecated.

50 of our tests are now invalid.  It tells me that we need a simple bit of sugar to allow an easy way to get going and then have the ExecutionInput as the way to do it in a more complex way.